android/src/main/kotlin/com/fingerprintjs/flutter/fpjs_pro/fpjs_pro_plugin/FpjsProPlugin.kt

```diff
@@ -128,4 +147,29 @@ fun parseRegion(region: String): Configuration.Region {
 
 const val INIT = "init"
 const val GET_VISITOR_ID = "getVisitorId"
-const val GET_VISITOR_DATA = "getVisitorData"
\ No newline at end of file
+const val GET_VISITOR_DATA = "getVisitorData"
+
+private fun getErrorCode(error: Error): String {
+  val errorType = when(error) {
+    is ApiKeyRequired -> "ApiKeyRequired"
+    is ApiKeyNotFound ->  "ApiKeyNotFound"
+    is ApiKeyExpired -> "ApiKeyExpired"
+    is RequestCannotBeParsed -> "RequestCannotBeParsed"
+    is Failed -> "Failed"
+    is RequestTimeout -> "RequestTimeout"
+    is TooManyRequest -> "TooManyRequest"
+    is OriginNotAvailable -> "OriginNotAvailable"
+    is HeaderRestricted -> "HeaderRestricted"
+    is NotAvailableForCrawlBots -> "NotAvailableForCrawlBots"
+    is NotAvailableWithoutUA -> "NotAvailableWithoutUA"
+    is WrongRegion -> "WrongRegion"
+    is SubscriptionNotActive -> "SubscriptionNotActive"
+    is UnsupportedVersion -> "UnsupportedVersion"
+    is InstallationMethodRestricted -> "InstallationMethodRestricted"
+    is ResponseCannotBeParsed -> "ResponseCannotBeParsed"
+    is NetworkError -> "NetworkError"
+    is UnknownError -> "UnknownError"
```

Comment:
Do we need this check here? Without it the `else` would handle it anyway, right?

Reply:
Fixed